catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemImpl.java

```diff
@@ -54,22 +54,6 @@
     protected String qualifier;
 
     /**
-     * An incoming content item whose ID will initially be
-     * <code>null</code> because the {@link ddf.catalog.CatalogFramework} will assign its GUID.
-     *
-     * @param byteSource      the {@link ContentItem}'s input stream containing its actual data
-     * @param mimeTypeRawData the {@link ContentItem}'s mime type
-     * @param filename        the {@link ContentItem}'s file name - can be null
-     * @param metacard        the {@link ContentItem}'s associated metacard
-     */
-    public ContentItemImpl(ByteSource byteSource, String mimeTypeRawData, String filename,
-            Metacard metacard) {
-        this(UUID.randomUUID()
-                .toString()
-                .replaceAll("-", ""), byteSource, mimeTypeRawData, filename, 0, metacard);
```

Comment:
I'm not sure; but i'd point out that this is on the impl, and not an API change.